lib/node_modules/@stdlib/math/base/special/sincos/lib/kernel_sincos.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+/*
+* The following copyright, license, and long comment were part of the original implementation available as part of [FreeBSD]{@link https://svnweb.freebsd.org/base/release/9.3.0/lib/msun/src/k_sin.c?view=co}.
```

Comment:
It's a fairly trivial blend of the two versions (from `sin` and `cos`) that are. A few multiplications are shared between the two so that I simply put them in the same function call. I've modified the comment to reflect that more precisely:

```
/*
* The following copyright and license were part of the original implementation available as part of FreeBSD implementations of [k_sin.c]{@link https://svnweb.freebsd.org/base/release/9.3.0/lib/msun/src/k_sin.c?view=co} and [k_cos.c]{@link https://svnweb.freebsd.org/base/release/9.3.0/lib/msun/src/k_cos.c?view=co}..
*
* The implementation follows the original sine and cosine kernels, but has been modified for JavaScript and combined into a single function.
*/
```